qa/tasks/cephfs/test_recovery_fs.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from os.path import join as os_path_join
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+class TestFSRecovery(CephFSTestCase):
+    """
+    Tests for recovering FS after loss of FSMap
+    """
+
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 3
+
+    def test_recover_fs_after_fsmap_removal(self):
+        data_pool = self.fs.get_data_pool_name()
+        metadata_pool = self.fs.get_metadata_pool_name()
+        # write data in mount
+        filedata, filename = 'some data on fs', 'file_on_fs'
+        filepath = os_path_join(self.mount_a.hostfs_mntpt, filename)
+        self.mount_a.write_file(filepath, filedata)
+        # Stop MDSs to allow removing the file system in the next step
+        self.fs.mds_stop()
```

Comment:
With the advent of cephadm, it's best to avoid starting/stopping MDS in tests when possible. I would suggest instead:

`self.fs.fail()`

Reply:
Done